src/Microsoft.DotNet.Build.CloudTestTasks/CreateAzureContainer.cs

```diff
@@ -114,51 +114,51 @@ public override bool Execute()
 
                     using (HttpResponseMessage response = await client.SendAsync(req))
                     {
-                        this.Log.LogMessage(
-                            MessageImportance.Normal,
+                        Log.LogMessage(
+                            MessageImportance.Low,
                             "Received response to create Container {0}: Status Code: {1} {2}",
-                            this.ContainerName, response.StatusCode, response.Content);
+                            ContainerName, response.StatusCode, response.Content.ToString());
+
+                        if (!response.IsSuccessStatusCode)
```

Comment:
This will trigger a failure if the container already exists which I don't believe is the behavior we want.  I'll get this fixed.